dpnp/backend/include/dpnp_gen_2arg_3type_tbl.hpp

```diff
@@ -66,7 +66,7 @@ MACRO_2ARG_3TYPES_OP(dpnp_fmod_c, cl::sycl::fmod((double)input_elem1, (double)in
 MACRO_2ARG_3TYPES_OP(dpnp_hypot_c, cl::sycl::hypot((double)input_elem1, (double)input_elem2), oneapi::mkl::vm::hypot)
 MACRO_2ARG_3TYPES_OP(dpnp_maximum_c, cl::sycl::max(input_elem1, input_elem2), oneapi::mkl::vm::fmax)
 MACRO_2ARG_3TYPES_OP(dpnp_minimum_c, cl::sycl::min(input_elem1, input_elem2), oneapi::mkl::vm::fmin)
-MACRO_2ARG_3TYPES_OP(dpnp_multiply_c, input_elem1* input_elem2, oneapi::mkl::vm::mul)
+MACRO_2ARG_3TYPES_OP(dpnp_multiply_array_array_c, input_elem1 * input_elem2, oneapi::mkl::vm::mul)
```

Comment:
we lost `dpnp_multiply_c` function here.

Reply:
also, `rng` might be a good example

Reply:
> we lost `dpnp_multiply_c` function here. I mean - it looks like we don't need this line of code here. Just delete it. Add dpnp_multiply_c as standalone function declaration. Also, it might be good to create separate `mathematical.hpp` file to collect all function related to this group there. The same as it is done at cython and python layer

Good idea. And it is better to do such refactoring work on separate PR.